### DISCUSSION TITLE
Previous Planning & Planning Part Identifiers (BT-125, BT-1251)

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#633] [Q&A] [answered]
Hi Lab !
can you please explain the meaning to me of the fields BT-125, BT-1251 ?
I mean, suppose that in a CN notice ( 16 ) we set the notice-id of a PIN notice (10) and its Lot (LOT-0001 ) 
In the dynamic validation, the CN Lot is compared with PIN Lot ( for simplicity let assume that both notice 16 and 10 has got only one Lot ) ? Does this happen only if both share the same BT-04 Procedure ? Or other ?
Thanks a lot
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
Form 10 is already a competition notice (PIN Call For Competition) and another competition notice for the same procedure wouldn't be expected.
Parts, that only exist in PIN Only notices (forms 4 to 6) may be combined to define lots. A same Part may also lead to different lots in different procedures.
For a given lot, BT-125 and BT-1251 are fields used to identify the PIN Only notices and their respective parts that lead to its definition.
Please note that PIN Only do not have an associated Procedure ID.
Also for non technical question, please contact the helpdesk instead.
KR

**Reply by rengit62** [reply]
OK thanks
But what do you mean, exactly for "PIN Only notices (forms 4 to 6) may be combined to define lots" ? 
Well suppose I make a notice type 4 with 2 parts, how these can be 'combined' to a Lot in notice 7 ?
Just repeating some parts values ? 
And another question, you wrote "For a given lot, BT-125 and BT-1251 are fields used to identify the PIN Only notices and their respective parts that lead to its definition". Is this done with dynamic validation ? And if the data do not correspond a validation error is thrown ?
Bye
Renato

**Reply by YvesJo** [reply]
Sorry, I forgot a comma and this should read: "Parts, that only exist in PIN Only notices (forms 4 to 6)**,** may be combined to define lots."
PIN Only notices contain "Parts" as opposition to other notices. The Parts are objects quite similar to the lots.
PIN Only and their Parts are about planned procurements over an extended period (e.g. 1 year). During that period multiple procedures (with Lots) will be launched. Lots may be composed combining element from different parts which may also belong to different PIN Only notices.
Please note that we currently do not have deep controls in place to check for consistency about the linked notices
